scripts/us_epa/superfund/site_contamination/process_sites_contamination.py

```diff
@@ -121,22 +130,31 @@ def write_sv_to_file(row, file_obj):
         row['value'] = True
         return row
 
-    except KeyError:
+    except:
```

Comment:
you should print out and inspect these errors instead of blindly handling them. there could be other errors...

Reply:
Added a print statement to indicate the cause of the exception.